screens/Profile.js

```diff
@@ -0,0 +1,70 @@
+import EStyleSheet from "react-native-extended-stylesheet";
+import Container from "../common/Container";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+
+
+
+
+function Profile({ navigation, route }) {
+
+    return (
+        <Container>
+            <View style={styles.back}>
+                <TouchableOpacity onPress={() => navigation.navigate("HomeScreen", route.params)}>
+                    <Image
+                        style={styles.image}
+                        source={require("../assets/back_arrow_icon.png")} />
+                </TouchableOpacity>
+            </View>
+            <View style={styles.container}>
+                <Text style={styles.text}>Profile</Text>
+                <Image source={require("../assets/profile_icon.png")} />
+            </View>
+            <View style={styles.first_info}>
+                <Text style={styles.text_info}>Name: {route.params.attributes.name}</Text>
```

Comment:
By using the Auth.currentUserInfo() function, you can get everything about a player (email, full name) without needing props.